src/pybamm_cookiecutter/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Copyright (c) 2023 Agriya Khetarpal. All rights reserved.
```

Comment:
Maybe we can attribute pybamm team here? Includes everyone!

Reply:
Fair enough!